src/lammps.cpp

```diff
@@ -48,7 +48,9 @@
 #include "version.h"
 #include "memory.h"
 #include "error.h"
-#include "mdi_init.h"
+#if defined(LMP_FIX_MDI_H)
```

Comment:
Sorry, but this is even worse. You must not depend on some include file being implicitly included. We may change the way how the lists of styles are processed (and there are some ideas about that being bounced around) and then this will fail. If anything, there should then be a global define added like `-DLMP_USER_MDI` like for other packages like `USER-OMP` or `USER-INTEL`.

Reply:
Okay, thanks for pointing me in the right direction!

Reply:
Am I correct to understand that I just need to sed `-DLMP_USER_MDI` into `Makefile.package`?  If so, 44cb26f should resolve this.

Reply:
Yes, but there is still the open question why this call to the library is needed here at all. From what I can tell this far, it would be sufficient to just have the same thing happen with a command style.